src/widgets/WebexMeeting/WebexMeeting.css

```diff
@@ -9,10 +9,16 @@
 .webex-meeting-widget__body {
   display: flex;
   flex: 1;
+  flex-flow: column;
+}
+
+.webex-meeting-widget__panel {
```

Comment:
can you try the following simplified structure:

.webex-meeting-widget
- body
  - meeting
  - rooster
- footer (or controls)

and, only if needed, a container for the spinner.